matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/room/RoomSyncHandler.kt

```diff
@@ -537,6 +539,11 @@ internal class RoomSyncHandler @Inject constructor(
         }
     }
 
+    private fun generateTimelineId(roomId: String, event: Event): String {
+        val threadIndicator = if (event.isThread()) "_thread_" else "_"
```

Comment:
why do we make a different timeline id between thread and normal?

Reply:
For the fact that it's technically a different timeline, that can contain the same events with the main timeline in parallel. But maybe re-using the main timelineId is enough, will update it